phpcs.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<ruleset name="Keboola">
+    <rule ref="PSR2"/>
+    <exclude-pattern>vendor/</exclude-pattern>
+    <config name="installed_paths" value="vendor/slevomat/coding-standard/"/>
+   <!-- <rule ref="SlevomatCodingStandard.TypeHints.TypeHintDeclaration">
```

Comment:
Jestli jsme rozhodnutí, že teda Slevomatí CS chceme, tak by bylo asi nejlepší z něj udělat package a tu jen odkazovat. Plus by se k tomu dal rovnou vždycky přihodit i CodeStyle.xml pro PhpStorm. Aby to bylo snadno použitelné. Takhle, když je to extra v každém projektu, tak se bojím, že bude pain to udržovat, jak říkal @odinuv. 

Mezitím jsem nicméně ještě narazil na https://github.com/keboola/http-extractor/issues/2#issuecomment-365631327

Reply:
ja jsem pro to Slevomati CS, udelejme ho jako package a ja to tady pak updatnu. btw. jak vidis zatim jsem ale par veci zakomentoval:) jak jsme se bavili

Reply:
on vlastne ten `cbf` asi funguje i na custom pravidla ze? Protoze jsem si vsimnul ze napr `Exception` catch mi to zmenilo na `Throwable`. Takze to pak zkusim jeste s povolenim tech pravidel

Reply:
Jo, některé ano (viz https://github.com/slevomat/coding-standard#sniffs-included-in-this-standard)